module/motion/QuinticWalk/src/WalkEngine.cpp

```diff
@@ -462,56 +455,60 @@ namespace module::motion {
 
         // Define trunk yaw target orientation position and velocity in euler angle and convertion to axis
         // vector
+        // const Eigen::Vector3f eulerAtSupport(0.0f,
+        //                                      params.trunk_pitch
+        //                                          + params.trunk_pitch_p_coef_forward * foot_step.getNext().x()
+        //                                          + params.trunk_pitch_p_coef_turn *
+        //                                          std::fabs(foot_step.getNext().z()), 0.5f * foot_step.getLast().z() +
+        //                                          0.5f * foot_step.getNext().z());
+        // const Eigen::Vector3f eulerAtNext(0.0f,
+        //                                   params.trunk_pitch
+        //                                       + params.trunk_pitch_p_coef_forward * foot_step.getNext().x()
+        //                                       + params.trunk_pitch_p_coef_turn * std::fabs(foot_step.getNext().z()),
+        //                                   foot_step.getNext().z());
+        // const Eigen::Matrix3f matAtSupport  = utility::math::euler::EulerIntrinsicToMatrix(eulerAtSupport);
+        // const Eigen::Matrix3f matAtNext     = utility::math::euler::EulerIntrinsicToMatrix(eulerAtNext);
+        // const Eigen::Vector3f axisAtSupport = Eigen::AngleAxisf(matAtSupport).axis();
+        // const Eigen::Vector3f axisAtNext    = Eigen::AngleAxisf(matAtNext).axis();
+
```

Comment:
Are we saving these for later or something? Might help to leave an explanation.

Reply:
Should never leave commented code in without a very very good reason